app/lib/api_docs/provider_data.rb

```diff
@@ -61,12 +61,21 @@ def service_ids
       end
     end
 
+    def service_metrics
+      @account.metrics.top_level
```

Comment:
```suggestion
      @account.metrics.includes(:owner).top_level
```

Because I see an N+1 coming below for this:
```ruby
METRIC_NAME = ->(metric) { "#{metric.friendly_name} | #{metric.owner.name}" }
	
def metric_names(metrics = self.metrics)
  metrics.map do |metric|
     { :name  => METRIC_NAME.call(metric),
```

Reply:
Btw, instead of `.metrics.top_level`, we can also do directly `.top_level_metrics` 😄